src/response/file.rs

```diff
@@ -219,31 +219,31 @@ impl Stream for FileStream {
     type Item = Result<Bytes, DynError>;
 
     fn poll_next(self: Pin<&mut Self>, context: &mut Context<'_>) -> Poll<Option<Self::Item>> {
-        let this = self.get_mut();
-
-        if this.remaining == 0 {
-            this.file = None;
-            this.buffer.fill(MaybeUninit::uninit());
+        if self.remaining == 0 {
```

Comment:
`remaining` is a Copy type and I prefer to have the preconditions happen immediately.